2021-09-06 22:09:27,128 INFO    MainThread:18000 [wandb_setup.py:_flush():69] setting env: {}
2021-09-06 22:09:27,128 INFO    MainThread:18000 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-06 22:09:27,128 INFO    MainThread:18000 [wandb_init.py:_log_setup():342] Logging user logs to D:\EPFL\pinet\wandb\run-20210906_220927-1aovsem8\logs\debug.log
2021-09-06 22:09:27,128 INFO    MainThread:18000 [wandb_init.py:_log_setup():343] Logging internal logs to D:\EPFL\pinet\wandb\run-20210906_220927-1aovsem8\logs\debug-internal.log
2021-09-06 22:09:27,129 INFO    MainThread:18000 [wandb_init.py:init():375] calling init triggers
2021-09-06 22:09:27,129 INFO    MainThread:18000 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-06 22:09:27,129 INFO    MainThread:18000 [wandb_init.py:init():424] starting backend
2021-09-06 22:09:27,129 INFO    MainThread:18000 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-09-06 22:09:27,132 INFO    MainThread:18000 [backend.py:ensure_launched():135] starting backend process...
2021-09-06 22:09:27,227 INFO    MainThread:18000 [backend.py:ensure_launched():139] started backend process with pid: 16940
2021-09-06 22:09:27,227 INFO    MainThread:18000 [wandb_init.py:init():429] backend started and connected
2021-09-06 22:09:27,228 INFO    MainThread:18000 [wandb_init.py:init():477] updated telemetry
2021-09-06 22:09:27,228 INFO    MainThread:18000 [wandb_init.py:init():500] communicating current version
2021-09-06 22:09:28,877 INFO    MainThread:18000 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-06 22:09:28,879 INFO    MainThread:18000 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-06 22:09:29,130 INFO    MainThread:18000 [wandb_init.py:init():540] starting run threads in backend
2021-09-06 22:09:32,552 INFO    MainThread:18000 [wandb_run.py:_console_start():1584] atexit reg
2021-09-06 22:09:32,553 INFO    MainThread:18000 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-09-06 22:09:32,553 INFO    MainThread:18000 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-09-06 22:09:32,556 INFO    MainThread:18000 [wandb_run.py:_redirect():1519] Redirects installed.
2021-09-06 22:09:32,556 INFO    MainThread:18000 [wandb_init.py:init():565] run started, returning control to user process
2021-09-06 22:09:33,474 INFO    MainThread:18000 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 1
2021-09-06 22:09:33,475 INFO    MainThread:18000 [wandb_run.py:_restore():1526] restore
2021-09-06 22:09:33,708 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 513
}

2021-09-06 22:09:34,041 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 513
}

2021-09-06 22:09:34,183 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 12284
}

2021-09-06 22:09:34,296 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 12310
}

2021-09-06 22:09:34,405 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 12310
}

2021-09-06 22:09:34,511 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 12310
  total_bytes: 12310
}

2021-09-06 22:09:34,616 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 12310
  total_bytes: 12310
}

2021-09-06 22:09:34,723 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 12310
  total_bytes: 12310
}

2021-09-06 22:09:34,831 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 12310
  total_bytes: 12310
}

2021-09-06 22:09:34,937 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 12310
  total_bytes: 12310
}

2021-09-06 22:09:35,394 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 12310
  total_bytes: 12310
}

2021-09-06 22:09:35,507 INFO    MainThread:18000 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 12310
  total_bytes: 12310
}

2021-09-06 22:09:36,709 INFO    MainThread:18000 [wandb_run.py:_show_files():1898] logging synced files
